config/application.yml.default

```diff
@@ -354,11 +354,11 @@ development:
   kantara_2fa_phone_restricted: true
   kantara_2fa_phone_existing_user_restriction: true
   kantara_restriction_enforcement_date: '2022-07-01'
-  irs_attempt_api_public_key: MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAyut9Uio5XxsIUVrXARqCoHvcMVYT0p6WyU1BnbhxLRW4Q60p+4Bn32vVOt9nzeih7qvauYM5M0PZdKEmwOHflqPP+ABfKhL+6jxBhykN5P5UY375wTFBJZ20Fx8jOJbRhJD02oUQ49YKlDu3MG5Y0ApyD4ER4WKgxuB2OdyQKd9vg2ZZa+P2pw1HkFPEin0h8KBUFBeLGDZni8PIJdHBP6dA+xbayGBxSM/8xQC0JIg6KlGTcLql37QJIhP2oSv0nAJNb6idFPAz0uMCQDQWKKWV5FUDCsFVH7VuQz8xUCwnPn/SdaratB+29bwUpVhgHXrHdJ0i8vjBEX7smD7pI8CcFHuVgACt86NMlBnNCVkwumQgZNAAxe2mJoYcotEWOnhCuMc6MwSj985bj8XEdFlbf4ny9QO9rETd5aYcwXBiV/T6vd637uvHb0KenghNmlb1Tv9LMj2b9ZwNc9C6oeCnbN2YAfxSDrb8Ik+yq4hRewOvIK7f0CcpZYDXK25aHXnHm306Uu53KIwMGf1mha5T5LWTNaYy5XFoMWHJ9E+AnU/MUJSrwCAITH/S0JFcna5Oatn70aTE9pISATsqB5Iz1c46MvdrxD8hPoDjT7x6/EO316DZrxQfJhjbWsCB+R0QxYLkXPHczhB2Z0HPna9xB6RbJHzph7ifDizhZoMCAwEAAQ==
+  irs_attempt_api_public_key: MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAvv5N+7C7OFM826bTIRqsrZt9sZVtoGcDLoPnw5euQIdNvTR56nWBl5y72WYvMilo+n2Au3xwVqGF0DQP5jaTzDoNZ7dr22SG4CIszDaYNPJQcGpNUPGFrHTEr8xXR7vqdZuU6/NBvTHe5zqgh4j69A91dwR7sel7h4KGcIdiEJ6O298gpWbIiH5y90BpLx1NjE9w0EktEqddKZxcNflB2re9j1QKtAITb7XmY9spaX1BRnExWIbZvYmrUJWJ0st/le4MFt5BnvQILhcdN2W4QIlUYQZz2101MXxyWkwpQGk6O3SFnvGUgkQyku6T+VRmNkNKLwUMAz2b0Pu6k+C37kkFvve101tlQkP2eQLS+H9RL3fEECJ/cz9qqVLPDRAFPkdFLA9fGuvxLSUIV5p9Hqf4h463+mcAPA6bYU7f0ifoobHZv7rGPw7uUg3n+zB3u9C2klOzHuivWlMSDPMsSftA59rpeTf0lLFzqfgSoXdxqLZdf/kgNd4NdSF8IABf7HF+mZ1Oo9BgOPWlwTBFDJkr6poN/NDDIOknVWCsyOq9v2HxvgF91ZSbub+P2pV4WywfwgUMLJL5N3IBlNjfFFpy5bH1JtkJu8F3wnHE1uKAOWj8iZFe5GSCltg6sUjyZCvE+o7aLlhzqcN/28M02J9321K1uB9Mw0N8kCGETKUCAwEAAQ==
   irs_attempt_api_public_key_id: key1
   irs_attempt_api_enabled: true
-  irs_attempt_api_auth_tokens: 'abc123'
-  irs_attempt_api_bucket_name: please-change-me
+  irs_attempt_api_auth_tokens: 'abc123,test-token-1,test-token-2'
+  liveness_checking_enabled: true
```

Comment:
ditto, do we want to commit all these changes?

Reply:
Sorry!  'twas accidental...